src/appleseed/renderer/kernel/rendering/pixelrendererbase.cpp

```diff
@@ -51,6 +51,7 @@ namespace renderer
 // PixelRendererBase class implementation.
 //
 
+const uint8 NoSampleHint = 0;
 const uint8 InvalidSampleHint = 1;
 const uint8 CorrectSampleHint = 2;
```

Comment:
Maybe it would be more logical that `CorrectSampleHint` is 0 and all problematic cases get non-zero values?

Also, can we find a better term than "hint"? These are not really hints, more like error codes...

Reply:
0 is because the tile is filled with 0 by default. This avoid to do extra process.

Reply:
Ok, makes sense.

The comment about "hint" stands though.